messages/snapshot.create.md

```diff
@@ -1,47 +1,37 @@
 # summary
 
-create a snapshot of a scratch org
+Create a snapshot of a scratch org.
 
 # description
 
-create a snapshot of a scratch org
 A snapshot is a point-in-time copy of a scratch org. The copy is stored in Salesforce and referenced by its unique name in a scratch org definition file.
 
-Use "sfdx force:org:snapshot:get" to get details, including status, about a snapshot creation request.
+Use "<%= config.bin %> org get snapshot" to get details, including status, about a snapshot creation request.
 
-To create a scratch org from a snapshot, include the "snapshot" entry (instead of "edition") in the scratch org definition file and set it to the name of the snapshot. Then use "sfdx force:org:create" to create the scratch org.
+To create a scratch org from a snapshot, include the "snapshot" entry (instead of "edition") in the scratch org definition file and set it to the name of the snapshot. Then use "<%= config.bin %> force:org:create" to create the scratch org.
```

Comment:
```suggestion
To create a scratch org from a snapshot, include the "snapshot" option (instead of "edition") in the scratch org definition file and set it to the name of the snapshot. Then use "<%= config.bin %> force:org:create" to create the scratch org.
```